modules/serverless-ossm-disabling-network-policies.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * /serverless/integrations/serverless-ossm-setup.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-ossm-disabling-network-policies.adoc_{context}"]
+= Disabling the default network policies 
+
+To disable the default network policy generation, you can annotate the `KnativeEventing` and `KnativeServing` custom resources (CRs) with the `serverless.openshift.io/disable-istio-net-policies-generation` annotation.
+
+This prevents the operator from applying the default network policies.
+
+.Prerequisites
+
+* You have installed the {ServerlessOperatorName} and Knative Serving and Knative Eventing on your cluster.
+* You have installed {SMProductName} with the mTLS functionality enabled.
+
+* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
+
+* Install the OpenShift CLI (`oc`).
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads.
+
+.Procedure
+
+. To disable the default network policies, annotate your Knative custom resources with `serverless.openshift.io/disable-istio-net-policies-generation` to `true`.
```

Comment:
You can consider the following example to represent this proc:

--------------------
. To disable the default network policies, add the  `serverless.openshift.io/disable-istio-net-policies-generation: "true"` annotation to your Knative custom resources.

.. `oc edit KnativeEventing` 

example output
 
apiVersion: operator.knative.dev/v1beta1
kind: KnativeEventing
metadata:
  name: knative-eventing
  namespace: knative-eventing
  annotations:
    serverless.openshift.io/disable-istio-net-policies-generation: "true"

.. `oc edit KnativeServing` 

Example output

apiVersion: operator.knative.dev/v1beta1
kind: KnativeServing
metadata:
  name: knative-serving
  namespace: knative-serving
  annotations:
    serverless.openshift.io/disable-istio-net-policies-generation: "true"

------------

if you want to keep this as a single-step proc then `.` should be `*` (bullet = unnumbered steps)
https://redhat-documentation.github.io/supplementary-style-guide/#single-step-procedures

Reply:
Good suggestion! Thank you! Done!